src/main/webapp/script.js

```diff
@@ -11,13 +11,19 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+let currentPage = 1;
+const postPerPage = 10;  // eslint-disable-line
```

Comment:
Why do we need to disable the lint for this line?

Reply:
It gives the error that it is not used in the script.